src/main/java/net/dv8tion/jda/api/entities/IPermissionContainer.java

```diff
@@ -27,7 +27,11 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-//TODO-v5: Need Docs
+/**
+ * Represents a {@link GuildChannel} that is capable of holding {@link PermissionOverride Permission Overrides}.
```

Comment:
We need to replace "holding" with something else.
I'm concerned that IPermissionContainer and IPermissionHolder will be confused. Not sure what to replace with yet tho, but we need to differentiate between the container being where the overrides exist on and a holder being like a "card holding member".